### ISSUE TITLE
Attempting to migrate from packages.config and there is no 'Migrate packages.config to PackageReference...' menu item

### ISSUE THREAD

**Issue by schellem** [NuGet/Home#6967]
Following directions from [here](https://docs.microsoft.com/en-us/nuget/reference/migrate-packages-config-to-package-reference)  to migrate from packages.config

VS UI

NuGet version (x.x.x.xxx): 4.6.1.5019

VS version (if appropriate): 2017 15.6.4

OS version (i.e. win10 v1607 (14393.321)): Win 10

1. Right click references and there is no 'Migrate packages.config to PackageReference...' menu item
2. Right click packages.config file and there is no 'Migrate packages.config to PackageReference...' menu item.

**Comment by mishra14** [comment]
@rohit21agrawal @jainaashish Do we have filtering on which projects get the option?

**Comment by rohit21agrawal** [comment]
This is a known issue and documented here: https://docs.microsoft.com/en-us/nuget/release-notes/nuget-4.7-rtm 

Try opening the Package Manager Console or Package Manager UI first, and then try again.

It might also be that the project you are trying to migrate is not supported (for eg. - asp.net, c++, sfproj)

**Comment by dasMulli** [comment]
> VS version (if appropriate): 2017 15.6.4

@schellem does it appear if you update your visual studio to a recent version? (15.7.2)

**Comment by rohit21agrawal** [comment]
@dasMulli thanks for pointing that out. This feature was released as part of 15.7, so won't be available in 15.6.4

**Comment by schellem** [comment]
Yes, I see the VS version I am using does not have this feature. All I saw(read) was this was for VS 2017 PREVIEW, to me when I see VS 2017 PREVIEW it means not yet released. I think using PREVIEW in a released product name is very misleading.

**Comment by dasMulli** [comment]
Created a PR to remove the `preview`: https://github.com/NuGet/docs.microsoft.com-nuget/pull/894

@schellem I see how this is confusing..
The way it currently works is: Visual Studio 2017 has multiple "channels" where you can get updates from.

Users can install a preview version of Visual Studio 2017 that gives them newer preview bits. Right now, 15.7.2 is the released Visual Studio 2017 version and the version "15.8 Preview 1.1" is the most recent preview version.
